releases/v20.2.0.md

```diff
@@ -0,0 +1,126 @@
+Core | **`ENUM` data types** | CockroachDB now supports the creation and management of [user-defined `ENUM` data types](../v20.2/enum.html) consisting of sets of enumerated, static values.
+Core | **Materialized Views** | CockroachDB now supports [materialized views](../v20.2/views.html#materialized-views), or views that store their selection query results on-disk.
+Core | **View Replacement** | CockroachDB now supports [replacing an existing view](../v20.2/create-view.html#replace-an-existing-view) with the `CREATE OR REPLACE VIEW` syntax.  
+Core | **Foreign Key Performance and Compatibility** | When adding the [`FOREIGN KEY`](../v20.2/foriegn-key.html) constraint, it is no longer required to have an index on the referencing columns.
+Core | **`EXPLAIN` Improvements** | The response of the `EXPLAIN` statement now includes the estimated number of rows scanned by the query as well as other usability improvements.
+Core | **Disallowing Full Table Scans** | You can use the new [`disallow_full_table_scans` session variable](..v20.2/set-vars.html) to disallow full table and secondary index scans.   
+
+### Recovery and I/O
+
+Version | Feature | Description
+--------|---------|------------
+Core | **Backup/Restore in Core Version** | The core version of CockroachDB now lets you perform full cluster backups, all restore options, as well as bulk exports. Incremental backups, locality-aware backups, and other advanced backup functionality continue to require an enterprise license.
+Enterprise | **KMS Support for Encrypted Backups** | You can now use [AWS Key Management Service (KMS)](../v20.2/take-and-restore-encrypted-backups.html#use-aws-key-management-service) to encrypt the files that full or incremental backups generate.
+Enterprise | **Backup Scheduling** | You can now [create schedules for CockroachDB backups](../v20.2/manage-a-backup-schedule#create-a-new-backup-schedule), as well as [view, pause, resume, and drop backup schedules](../v20.2/manage-a-backup-schedule#view-and-control-the-backup-schedule). Once a scheduled backup is created, you can use [`SHOW BACKUP`](../v20.2/manage-a-backup-schedule#view-scheduled-backup-details) to view the backup details. Note that backup scheduling for full backups is included in the core version of CockroachDB.  
+Core | **Import with User-Scoped Storage** | In addition to supporting bulk imports from cloud storage, CockroachDB now lets you [upload CSV files from your local machine to user-scoped file storage](../v20.2/use-userfile-for-bulk-operations.html) in your cluster. Once uploaded, a userfile can be reference by the `IMPORT` command to import data into a table. Userfiles can also be listed and deleted via CLI commands.
+Core | **Import with Default Expression** |
+
+### Deployment and Operations
+
+Version | Feature | Description
+--------|---------|------------
+N/A | Kubernetes Operator | NEED RYAN'S HELP TO SUMMARIZE THIS AND LINK TO THE RIGHT PLACE(S).
```

Comment:
@taroface, I need your help drafting this description and finding the right link(s).

Reply:
The CockroachDB Kubernetes Operator eases [deployment of secure CockroachDB clusters on Kubernetes](orchestrate-cockroachdb-with-kubernetes.html). The Operator can be used to create StatefulSets, authenticate pods, scale CockroachDB clusters, and perform rolling upgrades. 

cc @johnrk - does this line up with your sense of the "CockroachDB on Kubernetes" messaging?